### ISSUE TITLE
Is there a non-redraw mode?

### ISSUE THREAD

**Issue by awsdert** [Immediate-Mode-UI/Nuklear#118]
Basically I'm using this in an app that is supposed to run along side games and I don't want it causing slow downs because of it hogging resources, I'd like to know if there is a way to tell nuklear to only redraw on ui events like a mouse click and not on every frame unless a call to something like nkupdate()/nkredraw() is done
 
The app I'm making is this one:
https://github.com/awsdert/gasp

**Comment by dumblob** [comment]
Of course! That's the whole point of hackable libs with fine-grained options :wink:. Just look at the backends in `demo/...` directories and in each of them you'll see the main loop which is usually reacting only to input or other events (like some timeout or whatever). You can of course just render one image and quit if you want. There are endless possibilities :wink:.

**Comment by Hejsil** [comment]
Well, this is mostly something that can be solved outside of Nuklear. You already have to poll events from the backend so you can just initiate a redraw only when you see an event that actually would cause the ui to change (mouse clicks, window resize, keyboard events maybe).

You can also read the documentation for `NK_ZERO_COMMAND_MEMORY` which gives an example of how to avoid drawing the command buffer if it didn't change.
```
/// NK_ZERO_COMMAND_MEMORY  Defining this will zero out memory for each drawing 
/// command added to a drawing queue (inside nk_command_buffer_push). Zeroing command 
/// memory is very useful for fast checking (using memcmp) if command buffers are equal 
/// and avoid drawing frames when nothing on screen has changed since previous frame.
```
